[0.273s] Invoking command in '/home/masanao/ros2_ws/build/person_msgs': CMAKE_PREFIX_PATH=/home/masanao/ros2_ws/install/mypkg:/opt/ros/dashing /usr/bin/cmake /home/masanao/ros2_ws/src/person_msgs -DCMAKE_INSTALL_PREFIX=/home/masanao/ros2_ws/install/person_msgs
[0.290s] -- Found ament_cmake: 0.7.6 (/opt/ros/dashing/share/ament_cmake/cmake)
[0.319s] -- Found PythonInterp: /usr/bin/python3 (found suitable version "3.6.9", minimum required is "3") 
[0.320s] -- Using PYTHON_EXECUTABLE: /usr/bin/python3
[0.541s] -- Found rosidl_default_generators: 0.7.0 (/opt/ros/dashing/share/rosidl_default_generators/cmake)
[1.842s] -- Found rosidl_adapter: 0.7.10 (/opt/ros/dashing/share/rosidl_adapter/cmake)
[2.978s] -- Found PythonInterp: /usr/bin/python3 (found version "3.6.9") 
[3.111s] CMake Error at /opt/ros/dashing/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:60 (message):
[3.111s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[3.112s]   person_msgs --arguments-file
[3.112s]   /home/masanao/ros2_ws/build/person_msgs/rosidl_adapter__arguments__person_msgs.json
[3.112s]   --output-dir
[3.112s]   /home/masanao/ros2_ws/build/person_msgs/rosidl_adapter/person_msgs
[3.112s]   --output-file
[3.112s]   /home/masanao/ros2_ws/build/person_msgs/rosidl_adapter/person_msgs.idls)
[3.113s]   returned error code 1:
[3.113s] 
[3.113s]   Error processing 'unit8 age' of 'person_msgs/Person': 'unit8'
[3.113s] 
[3.113s]   Traceback (most recent call last):
[3.113s] 
[3.113s]     File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
[3.113s]       "__main__", mod_spec)
[3.114s]     File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
[3.114s]       exec(code, run_globals)
[3.114s]     File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
[3.114s]       sys.exit(main())
[3.114s]     File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_adapter/main.py", line 55, in main
[3.114s]       pathlib.Path(relative_path), output_dir)
[3.114s]     File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_adapter/__init__.py", line 20, in convert_to_idl
[3.114s]       package_dir, package_name, interface_file, output_dir / 'msg')
[3.114s]     File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[3.114s]       msg = parse_message_string(package_name, input_file.stem, content)
[3.114s]     File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_adapter/parser.py", line 504, in parse_message_string
[3.114s]       Type(type_string, context_package_name=pkg_name),
[3.114s]     File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_adapter/parser.py", line 273, in __init__
[3.114s]       context_package_name=context_package_name)
[3.115s]     File "/opt/ros/dashing/lib/python3.6/site-packages/rosidl_adapter/parser.py", line 202, in __init__
[3.115s]       raise InvalidResourceName(self.type)
[3.115s] 
[3.115s]   rosidl_adapter.parser.InvalidResourceName: unit8
[3.115s] 
[3.115s] Call Stack (most recent call first):
[3.115s]   /opt/ros/dashing/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[3.115s]   CMakeLists.txt:24 (rosidl_generate_interfaces)
[3.115s] 
[3.115s] 
[3.115s] -- Configuring incomplete, errors occurred!
[3.116s] See also "/home/masanao/ros2_ws/build/person_msgs/CMakeFiles/CMakeOutput.log".
[3.119s] Invoked command in '/home/masanao/ros2_ws/build/person_msgs' returned '1': CMAKE_PREFIX_PATH=/home/masanao/ros2_ws/install/mypkg:/opt/ros/dashing /usr/bin/cmake /home/masanao/ros2_ws/src/person_msgs -DCMAKE_INSTALL_PREFIX=/home/masanao/ros2_ws/install/person_msgs
